ktor-utils/common/src/io/ktor/util/CaseInsensitiveMap.kt

```diff
@@ -41,7 +41,7 @@ class CaseInsensitiveMap<Value> : MutableMap<String, Value> {
         )
 
     override val entries: MutableSet<MutableMap.MutableEntry<String, Value>>
-        get() = DelegatingMutableSet<MutableMap.MutableEntry<CaseInsensitiveString, Value>, MutableMap.MutableEntry<String, Value>>(
+        get() = DelegatingMutableSet(
```

Comment:
How this change relates to explicit API mode?

Reply:
It's code style change. Are we ok to keep it here?

Reply:
No